recipe/migrations/aws_c_io0913.yaml

```
@@ -0,0 +1,7 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
```

All feedstocks in the rebuild are maintained by me.
```suggestion
  migration_number: 1
  automerge: true
```